cobalt2.tmTheme

```diff
@@ -876,8 +876,8 @@
      <key>scope</key>
      <string>
        , string.unquoted.label.js,
-       , meta.object-literal.key.js -string,
```

Comment:
FYI, this will probably change how some text is colored in Sublime.

Atom and VS Code don't support negation and there was no other way I saw to write another selector that would hit the same things